src/pages/workspaces/List.js

```diff
@@ -499,7 +526,13 @@ export const WorkspaceList = () => {
         workspace: getWorkspace(requestingAccessWorkspaceId),
         onDismiss: () => setRequestingAccessWorkspaceId(undefined)
       }),
-      loadingWorkspaces && (!workspaces ? transparentSpinnerOverlay : topSpinnerOverlay)
+      loadingWorkspaces && (!workspaces ? transparentSpinnerOverlay : topSpinnerOverlay),
+      !loadingWorkspaces && anyPersistentDiskRequiresMigration(workspaces, gcpPersistentDisks) && h(UnboundDiskNotification, {
```

Comment:
There doesn't appear to be any way to get keyboard focus into this notification to interact with the link.

Reply:
OK, how would I do that?

Reply:
I would recommend that you just put it directly inline in the page. Perhaps under "Dedicated spaces for you and…"?

Reply:
I'm following the UX mockup for the banner in the ticket. Are you suggesting not doing that?

Reply:
It has to be keyboard accessible, and given the relatively small audience and the fact that this feature will be short-lived, I was offering a suggestion that I thought would be the simplest to implement.

But certainly it is worth checking with UX to see if they are OK with it being inline, if you don't wish to figure out how to get the notification into the page's natural tab ordering.

Reply:
It might be enough to just move it up in the HTML. Perhaps if I could get to the end of the workspaces list it would actually then go into the notification?

Reply:
Ah yes, it goes into the notification after navigating through the last workspace in the list. So insert the HTML up at the top of the page.

Reply:
Where should the banner go then? after the top bar? in the "main" hierarchy?